frontend_scv/lib/widgets/agreement/contract_action_area.dart

```diff
@@ -59,17 +59,28 @@ class _ContractActionAreaState extends State<ContractActionArea> {
   @override
   Widget build(BuildContext context) {
     //if seal agreement button must be shown or not:
+    const double _borderWidth = 1;
+
     if (_isLoading) {
       return AwesomeLoader(
         loaderType: AwesomeLoader.AwesomeLoader4,
         color: Colors.pink,
       );
     }
 
-    if (widget._agreement.movedToBlockchain == false ||
+    if (widget._agreement.movedToBlockchain ==
+            false || //TODO: Ronan thinks Kevin should change this (Maybe to a null check?)
         widget._agreement.blockchainId == BigInt.from(-1)) {
+      //TODO: The second condition will always fail, an exception will be thrown since Bigints are always positive
```

Comment:
Has the todo been solved or implemented?

Reply:
No.